main_test.go

```diff
@@ -33,6 +70,17 @@ func TestRunme(t *testing.T) {
 	})
 }
 
+func TestRunmeFilePermissions(t *testing.T) {
+	if isDocker() {
```

Comment:
Consider moving this test case to a new file and use tags to toggle inclusion/exclusion from docker? perhaps i'm overlooking something.

Reply:
lemme ask a question: why is this `isDocker` check required? do these tests ever run in docker?

Reply:
@sourishkrout, For some reason, I don't know why the **chmod** command is not working. It is related to the volume. So I have to check if it's running inside a container to skip it. It is passing successfully locally and inside Github Actions Runner.

Reply:
I can replicate different results when running this test in Docker and locally. I suspect that the Docker image does not create a user and runs everything as root which results in no permission denied error even for the `000` permissions.

Either try that or control this with build tags. If the latter, it's a bit complicated... We already have a tag `test_with_docker`. It means that the file, like `command_docker_test.go` should be run only in the environment that provides Docker. We could introduce another tag which would indicate that the test should not in a Docker environment. It will require setting it and passing properly. I would try the option with a non-root user first.